src/Settings/AdvancedParameters.as

```diff
@@ -24,6 +24,26 @@ namespace PluginSettings
         RMC_MX_Url = UI::InputText("MX Base URL", RMC_MX_Url);
         UI::SetPreviousTooltip("Use this URL for API calls to ManiaExchange. Useful for hosting your own service for caching and preloading API responses for better performance.\nOnly change if you know what you're doing!");
 
+#if TMNEXT
+        if (UI::Button("Use official TMX API")) {
+            RMC_MX_Url = "https://" + MX_URL;
```

Comment:
is this needed, use default should reset to this?

Reply:
Reset to default does, but also resets other things.

Reply:
I thought it'd be better to have a button that explicitly does just that, both for consistency (there will be other buttons that set the URL), and because it's more obvious from a UX point of view how to undo changing the URL without affecting other settings.

Reply:
i dont agree nor disagree on your assessment, but am fine with keeping it